docs/pages/access-controls/login-rules/reference.mdx

```diff
@@ -585,6 +586,82 @@ Expression | Result
 `strings.replaceall(set("user-name"), "-", "_")` | `("user_name")`
 `strings.replaceall(set("user-alice", "user-bob"), "user-", "")` | `("alice", "bob")`
 
+### `email.local`
+
+<Admonition type="note">
+The `email.local` helper was introduced in Teleport v13.3.0, all Auth servers
+must be running this version or greater before it can be used.
```

Comment:
```suggestion
The `email.local` helper was introduced in Teleport v13.3.0. All Auth Service instances
must be running this version or greater before it can be used.
```